platform/linux-dpdk/pool/dpdk.c

```diff
@@ -91,12 +90,12 @@ int odp_pool_init_global(void)
 	return 0;
 }
 
-int odp_pool_init_local(void)
+static int dpdk_pool_init_local(void)
 {
```

Comment:
This is the empty function that you have commented above about making it NULL or just return true all ways. This is needed here to skip compiler issue.